pkg/api/v1beta1/types.go

```diff
@@ -53,13 +53,13 @@ type ContainerManifest struct {
 	Version string `json:"version" description:"manifest version; must be v1beta1"`
 	// Required: This must be a DNS_SUBDOMAIN.
 	// TODO: ID on Manifest is deprecated and will be removed in the future.
-	ID string `json:"id" description:"manifest name; must be a DNS_SUBDOMAIN"`
+	ID string `json:"id" description:"manifest name; must be a DNS_SUBDOMAIN; cannot be updated"`
 	// TODO: UUID on Manifext is deprecated in the future once we are done
 	// with the API refactory. It is required for now to determine the instance
 	// of a Pod.
-	UUID          types.UID     `json:"uuid,omitempty" description:"manifest UUID, populated by the system, read-only"`
+	UUID          types.UID     `json:"uuid,omitempty" description:"manifest UUID, populated by the system, read-only; cannot be updated"`
```

Comment:
I don't think we need to specify "cannot be updated" for fields that are already "read-only", but I won't block merging on that.